### ISSUE TITLE
Add callback for unload frame?

### ISSUE THREAD

**Issue by rmunch** [fancyapps/fancyBox#749]
For multi-frame Fancybox galleries, it might be helpful if there were a callback that is called just before a frame unloads (e.g. removed from DOM, etc.) in order to handle custom unbinding / memory cleanup.

I ran into a gotcha with my gallery where I needed to do manual cleanup of the previous frame during the beforeShow callback. This worked okay when switching frames (the previous frame is offscreen when unbinding, etc.), but I had a memory leak because I forgot to run this in the beforeClose callback too.

So basically this is possible as-is but is a bit circuitous to get working because you have to run the unbind code from both the beforeClose and beforeShow, and you also have to conditionalize the code in the beforeClose callback so it doesn't run when there is no previous frame (in the case of the first frame).

**Comment by fancyapps** [comment]
Hi, please, upgrade to v3. It has "onReset" event that is triggered for each slide before cleaning up.
